lib/sda/host_test.go

```diff
@@ -250,7 +250,7 @@ func TestTreePropagation(t *testing.T) {
 		network.DumpTypes()
 		t.Fatal("h1 didn't receive SendTree type:", msg.MsgType)
 	}
-	if msg.Msg.(sda.TreeMarshal).EntityId != uuid.Nil {
+	if msg.Msg.(sda.TreeMarshal).EntityListID != sda.EntityListID(uuid.Nil) {
```

Comment:
mhhh.. Is there not a way to remove some noise in these checks ? 
Having some kind of generic checker for specific UUID ?
 Along the lines of ``` EntityListID.Equal(uuid.Nil) ``` 
I don't know I haven't tried it but *if* this is possible that could be a real **plus** .